src/server/jobs/new-order-jobs.ts

```diff
@@ -191,20 +198,23 @@ const newOrderDailyReset = createJob('new-order-daily-reset', '0 0 * * *', async
       WHERE "NewOrderPlayer"."userId" = affected."userId"
     `;
 
+    // Update counters
+    await Promise.all(
+      batchWithFervor.map((j) => {
+        const { userId, exp, correctJudgments, totalJudgments, fervor } = j;
+        return Promise.all([
+          expCounter.increment({ id: userId, value: exp }),
+          correctJudgmentsCounter.increment({ id: userId, value: correctJudgments }),
+          allJudgmentsCounter.increment({ id: userId, value: totalJudgments }),
+          fervorCounter.increment({ id: userId, value: fervor }),
```

Comment:
The whole point of the reset is re-creating this data, we shouldn't increment it.

Reply:
You should update the counters themselves so that they can fetch the initial user's data from Clickhouse